infrastructure/lib/atat-web-api-stack.ts

```diff
@@ -21,12 +20,14 @@ export class AtatWebApiStack extends cdk.Stack {
     this.templateOptions.description = "Resources to support the ATAT application API";
 
     // Create a shared DynamoDB table that will be used by all the functions in the project.
-    const table = new dynamodb.Table(this, "AtatTable", {
-      partitionKey: { name: "id", type: dynamodb.AttributeType.STRING },
-      billingMode: dynamodb.BillingMode.PROVISIONED,
-      readCapacity: 1,
-      writeCapacity: 1,
-      removalPolicy: props?.removalPolicy,
+    const { table } = new PITRTable(this, "AtatTable", {
+      tableProps: {
+        partitionKey: { name: "id", type: dynamodb.AttributeType.STRING },
+        billingMode: dynamodb.BillingMode.PROVISIONED,
+        readCapacity: 1,
+        writeCapacity: 1,
```

Comment:
While we're here, should we consider flipping to On Demand? We had previously done PROVISIONED to have predictable, static cost. But I think with our usage patterns, On Demand would be cheaper.

Reply:
That makes sense. I thought this might have been previously decide, so I didn't change anything.